.gitignore

```diff
@@ -43,9 +43,20 @@ local_test/*
 !local_test/index.html
 local_test/js/*
 local_test/css/*
+# folder of projects to test 
+local_test/intesting
```

Comment:
> But I do think we should add an item to the PR checklist to make sure we revert the index.html file to its original state.

@tchaffee agreed - been meaning to do this for a while. Especially important as more people begin contributing.